yang-module-versioning/draft-ietf-netmod-yang-module-versioning.xml

```diff
@@ -400,14 +396,12 @@ module: ietf-yang-library-revisions
           <t>Servers SHOULD set both the "deprecated-nodes-implemented" and "obsolete-nodes-absent" leafs to "true".</t>
           <t>If a server does not set the "deprecated-nodes-implemented" leaf to "true", then clients MUST NOT rely solely on the "rev:non-backwards-compatible" statements to determine whether two module revisions are backwards-compatible, and MUST also consider whether the status of any nodes has changed to "deprecated" and whether those nodes are implemented by the server.</t>
         </section>
-      </section>
     </section>
     <section anchor="guidelines" title="Guidelines for using the YANG module update rules">
       <t>The following text updates section 4.7 of
         <xref target="RFC8407"/> to revise the guidelines for updating YANG modules.
       </t>
       <section anchor="guidelines_authors" title="Guidelines for YANG module authors">
-        <t>All IETF YANG modules MUST conform to this specification.  In particular, sections: <xref target="yang_revision_handling"/>, <xref target="import-overview"/>, and the guidelines documented in this section.</t>
```

Comment:
Feb 13th meeting: this stays since this is not for yang-semver. See if this doc says anywhere that IETF modules must use semver, and if so remove.

Reply:
Didn't see any instance where this doc says to use semver.